internal/mage/sdk/python.go

```diff
@@ -0,0 +1,107 @@
+package sdk
+
+import (
+	"context"
+	"os"
+	"strings"
+
+	"dagger.io/dagger"
+	"github.com/dagger/dagger/internal/mage/util"
+	"github.com/magefile/mage/mg"
+)
+
+const (
+	pythonGeneratedAPIPath = "sdk/python/src/dagger/codegen.py"
```

Comment:
This is not the generated file:
```toml
generate = "dagger-py generate --output src/dagger/api/gen.py"
```

Reply:
With https://github.com/dagger/dagger/pull/3718 there are two generated files: `gen.py` and `gen_sync.py`:

```toml
generate = [
     "dagger-py generate --output src/dagger/api/gen.py",
     "dagger-py generate --output src/dagger/api/gen_sync.py --sync",
 ]
```

Reply:
Fixed